setup.py

```diff
@@ -29,11 +29,8 @@
     # Nucleus needs uptodate protocol buffer compiler (protoc).
     'protobuf>=3.6.1',
     'mmh3<2.6',
-    # Need to explicitly install v<=1.14.0. apache-beam requires
-    # google-cloud-pubsub 0.39.1, which relies on google-cloud-core<0.30dev,
-    # >=0.29.1. google-cloud-storage also has requirements on google-cloud-core,
-    # and version 1.14.0 resolves the dependency conflicts.
-    'google-cloud-storage<=1.14.0'
+    'google-cloud-storage',
```

Comment:
Is this still needed?

Reply:
Yes, we need this package for GCS usage.